dipy/denoise/tests/test_ascm.py

```diff
@@ -105,7 +105,7 @@ def test_ascm_accuracy():
 
     # the test data was constructed in this manner
     mask = test_data > 50
-    sigma = estimate_sigma(test_data, N=4)
+    sigma = estimate_sigma(test_data, N=4).item()
```

Comment:
if you do that, you need to update this use of `sigma`: https://github.com/dipy/dipy/blob/9731f0cb7d2a5967bdfc2490aea98b4abc0da3d9/dipy/denoise/tests/test_ascm.py#L127

CI`s will fail with the current change

Reply:
Yep, sorry for the oversight. Done in 8336b65.